src/graph/interface/op_schema.hpp

```diff
@@ -167,11 +167,12 @@ class op_schema_t {
     op_schema_t &set_output(size_t out_offset, std::string &&out_name,
             std::string &&dtype_string = "any");
 
-    /*! @brief Enable commutative inputs */
-    op_schema_t &set_commutative_inputs();
+    /*! @brief Enable commutative inputs for specific inputs*/
+    op_schema_t &set_commutative_inputs(
+            const std::set<size_t> &inputs = {0, 1});
 
     /*! @brief Get whether the commutative inputs option is enabled or not */
-    bool get_commutative_inputs() const;
+    bool get_commutative_inputs(const std::set<size_t> &inputs = {}) const;
```

Comment:
Add a brief description for this function, check if the inputs set are all commutative inputs, if inputs is an empty set, check if the op has commutative inputs.

Reply:
Updated

Reply:
The default argument here is confusing.

Reply:
The default argument here is to maintain compatibility with zero-argument usage(in [nested_matcher.cpp#L73](https://github.com/uxlfoundation/oneDNN/blob/0fc69752f83811c008847ee1ee73076a70244dad/src/graph/utils/pm/nested_matcher.cpp#L73)), where we simply check if the op has commutative inputs. It might make sense to split this into two separate functions:

- is_commutative_inputs(inputs)
- is_commutative_op()

Reply:
Updated